avaframe/out3Plot/outAIMEC.py

```diff
@@ -571,17 +552,43 @@ def resultVisu(cfgPath, rasterTransfo, resAnalysis, plim):
             if mk == len(markers):
                 mk = 0
         ax1.legend(loc='lower left')
+
     plt.xlim([-0.01, max(1, max(rFP))])
     plt.ylim([0, 1.01])
     plt.grid('on')
 
     outFileName = ''.join([cfgPath['dirName'], '_dptr', str(int(plim)), '_ROC'])
-    outname = os.path.join(cfgPath['pathResult'], 'pics', outFileName)
 
-    if not os.path.exists(os.path.dirname(outname)):
-        os.makedirs(os.path.dirname(outname))
-    fig.savefig(outname)
+    _saveAndOrPlot(cfgPath, cfgFlags, outFileName, fig)
+
+    return
+
+
+def _saveAndOrPlot(cfgPath, cfgFlags, outFileName, fig):
+    """
+    Receive a plot handle and config and check whether to save and or plot
+    """
+
+    if cfgFlags.getboolean('savePlot'):
+        outname = os.path.join(cfgPath['pathResult'], 'pics', outFileName)
+        if not os.path.exists(os.path.dirname(outname)):
+            os.makedirs(os.path.dirname(outname))
+        fig.savefig(outname)
+
+    if cfgFlags.getboolean('plotFigure'):
+        plt.show()
+    else:
+        plt.ioff()
```

Comment:
could we add this in the plot utils instead ?

Reply:
Good idea! Will add this